examples/twittermap/web/public/javascripts/map/controllers.js

```diff
@@ -642,6 +672,14 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
             style: $scope.styles.countyUpperStyle
           });
           setCenterAndBoundry($scope.geojsonData.county.features);
+          if ($scope.status.logicLevel === 'county'){
+            $scope.map.addLayer($scope.polygons.stateUpperPolygons);
+            $scope.map.addLayer($scope.polygons.countyPolygons);
+            if (cloudberry.parameters.maptype === "countmap"){
+              cloudberry.parameters.geoLevel = $scope.status.logicLevel;
+              cloudberry.query(cloudberry.parameters);
+            }
+          }
```

Comment:
We don't need to consider "statePolygons" or "cityPolygons" at this time?

Reply:
This is called when county.json file is successfully loaded. "statePolygons" layer should be added when state.json is loaded and $scope.status.logicLevel === 'state'. If the logic level is city, both state and county layers will not be added. Instead, I call loadCityJsonByBound(onEachFeature) to add city polygon layer.

Reply:
Got it.